datapower-docker-compose-dp-mq/src/datapower/Dockerfile~

```diff
@@ -0,0 +1,8 @@
+FROM ibmcom/datapower:beta
+
+ENV DATAPOWER_ACCEPT_LICENSE=true \
+    DATAPOWER_INTERACTIVE=true \
+    DATAPOWER_WORKER_THREADS=4 
+COPY src/ /drouter/
+EXPOSE 9090 1707
```

Comment:
I assume you had port 9090 here for debugging purposes. I don't think we want to expose 9090 for the tutorial as we want to show 'best-practices'